build-system/eslint-rules/html-template.js

```diff
@@ -181,7 +181,7 @@ function create(context) {
     const matches = [];
 
     let match;
-    while ((match = invalid.exec(string))) {
+    while ((match = invalid.exec(string)) && !string.includes('<svg')) {
```

Comment:
Why not adding `svg` to line 180? That is essentially an allowlist.

Reply:
Yep, good idea, I have implemented this. Please review again.